docs/english/concepts/message-sending.md

```diff
@@ -38,4 +39,60 @@ def show_datepicker(event, say):
             blocks=blocks,
             text="Pick a date for me to remind you"
         )
-```
\ No newline at end of file
+```
+
+## Streaming messages {#streaming-messages}
+
+You can have your app's messages stream in to replicate conventional AI chatbot behavior. This is done through three Web API methods:
+
+* [`chat_startStream`](/reference/methods/chat.startstream)
+* [`chat_appendStream`](/reference/methods/chat.appendstream)
+* [`chat_stopStream`](/reference/methods/chat.stopstream)
+
+The Python Slack SDK provides a [`chat_stream()`](https://docss.slack.dev/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility to streamline calling these methods. Here's an excerpt from our [Assistant template app](https://github.com/slack-samples/bolt-python-assistant-template/blob/main/listeners/assistant/assistant.py)
```

Comment:
```suggestion
The Python Slack SDK provides a [`chat_stream()`](/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility to streamline calling these methods. Here's an excerpt from our [Assistant template app](https://github.com/slack-samples/bolt-python-assistant-template)
```

🐍 suggestion: Removing perhaps more ambiguous links for the exploring readers!

Reply:
🗣️ note: I understand the linked file of the sample exists on a branch at the moment, but I'm not certain that we will keep this file structure into the future after this lands. Hoping to avoid updates later with this!